app/Http/Controllers/ReportGroupController.php

```
@@ -65,37 +75,64 @@ public function show(string $id)
      */
     public function edit(string $id)
     {
-        //
     }
 
     /**
      * Update the specified resource in storage.
      */
-    public function update(Request $request, string $id)
+    public function update(ReportGroupStoreRequest $request, string $id)
```

PHPMD (2.15.0)

Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)